libvast/src/format/syslog.cpp

```diff
@@ -0,0 +1,126 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/format/syslog.hpp"
+
+#include "vast/error.hpp"
+#include "vast/fwd.hpp"
+#include "vast/type.hpp"
+
+#include <caf/expected.hpp>
+
+namespace vast {
+namespace format {
+namespace syslog {
+
+namespace {
+
+// At the moment map_type cannot be indexed, therefore it is not part of the
+// schema for now
+type make_syslog_msg_type() {
+  // clang-format off
+  return record_type{{
+    {"facility", count_type{}},
+    {"severity", count_type{}},
+    {"version", count_type{}},
+    {"ts", time_type{}.attributes({{"timestamp"}})},
+    {"hostname", string_type{}},
+    {"app_name", string_type{}},
+    {"process_id", string_type{}},
+    {"message_id", string_type{}},  
+    // {"structered_data", map_type{
+    //   string_type{}.name("id"),
+    //   map_type{string_type{}.name("key"), string_type{}.name("value")}.name("params")},
+    // },
+    {"message", string_type{}},
+  }}.name("syslog::msg");
```

Comment:
```suggestion
  }}.name("syslog");
```

I think the `::msg` is unnecessary and just leads to confusion. It shouldn't be needed.